libbpf-tools/memleak.c

```diff
@@ -1041,8 +1051,14 @@ int attach_uprobes(struct memleak_bpf *skel)
 	ATTACH_UPROBE_CHECKED(skel, realloc, realloc_enter);
 	ATTACH_URETPROBE_CHECKED(skel, realloc, realloc_exit);
 
-	ATTACH_UPROBE_CHECKED(skel, mmap, mmap_enter);
-	ATTACH_URETPROBE_CHECKED(skel, mmap, mmap_exit);
+	// third party alloctor like jemallloc not support mmap, so remove the check.
```

Comment:
alloctor => allocator

Please don't use C++ style comment.

Reply:
> alloctor => allocator
> 
> Please don't use C++ style comment.

you mean use /* third party allocator like jemallloc not support mmap, so remove the check. */ ?

Reply:
> alloctor => allocator
> 
> Please don't use C++ style comment.
done